README.md

```diff
@@ -23,21 +23,21 @@ You can then use Chronos:
 <?php
 require 'vendor/autoload.php';
 
-use Cake\Chronos\Chronos;
+use Cake\Chronos\DateTime;
 
-printf("Now: %s", Chronos::now());
+printf("Now: %s", DateTime::now());
 ```
```

Comment:
Now let's add some real arguments. Chronos is a sexeh name whereas Datetime us not :troll:.

(I don't like a custom sexeh well made time lib masquerading as a php std lib)